Automattic/pocket-casts-android
# End of Year: Show first and last story

| 📘 Project: #410 |
|:---:|

iOS PR: https://github.com/Automattic/pocket-casts-ios/pull/420

## Description
Adds the first and last story. The last story also adds a replay option.

## Testing Instructions
1. Set `END_OF_YEAR_ENABLED` feature flag to `true` in `base.gradle`
2. Run the app and make sure you're logged in to an account with a few items on Listening History
3. When the prompt appears, tap "View My 2022"
4. ✅ Check that the first story contains a "Let's celebrate..." string
5. Tap on the right side to go to the next story
6. ✅ Check that it works
7. Advance to the last story
8. ✅ Wait for it to end, notice that the stories won't start over
9. Tap the "Replay" button
10. ✅ Check that the stories start again

## Screenshots or Screencast 

| Intro story | Epilogue story |
| ---------- | ---------- |
<img width=320 src="https://user-images.githubusercontent.com/1405144/199909909-f16e12e7-759e-46ee-a611-99b1a6e8bbbb.png"/> | <img width=320 src="https://user-images.githubusercontent.com/1405144/199909927-5bdfe58a-786d-4d6e-bed6-10a55488f55d.png"/>

## Checklist
 
#### I have tested any UI changes...
